src/main/java/org/example/controllers/JobRoleController.java

```diff
@@ -139,6 +149,54 @@ public Response getJobRoleById(@PathParam("jobRoleId") final int jobRoleId) {
         }
     }
 
+    @POST
+    @Produces(MediaType.APPLICATION_JSON)
+    @Consumes(MediaType.MULTIPART_FORM_DATA)
+    @RolesAllowed({UserRole.ADMIN, UserRole.USER})
+    @ApiOperation(
+            value = "Sends CV pdf to S3 bucket and application information in the database",
```

Comment:
Sends CV pdf to S3 bucket and saves application information in the database
OR
Sends CV pdf to S3 bucket and application information to the database"